utils/shonenjump/patches/0001-store.go-do-not-call-Age.patch

```diff
@@ -0,0 +1,11 @@
+--- a/jump/store.go
++++ b/jump/store.go
```

Comment:
Patch will be upstreamed or it was already send to upstream?

Reply:
Upstream is thinking about how to handle the code, see: https://github.com/suzaku/shonenjump/issues/10

Haven't seen anything in a while.  If you want, I will remove that patch and build pure upstream.

Reply:
Okay, from your patch it is not clear for me (as I am not involved either using shonenjump anyhow) that it was related to some issue, which you are dealing with. Could you please improve this patch to be more upstreamable based on wiki - https://openwrt.org/submitting-patches ?

So it has Git header - author, commit subject, description. Otherwise, we would end up with the situation as it is in the main repository where these patches are, but no author, no description and we can just guess what they are doing.

Reply:
Done, let me know if it is acceptable.  Will keep this unresolved until I hear back.

Reply:
Thanks, much better. 👍

Reply:
Great, thanks for you help to review.